mne_neuron/network.py

```diff
@@ -258,6 +260,28 @@ def gid_to_type(self, gid):
             if gid in gids:
                 return gidtype
 
+    def reset_src_event_times(self, seed=None, debug=False, inc_evinput=0.0):
```

Comment:
I would remove these parameters and simplify the code below so that the bare minimum remains needed to reproduce without the unnecessary if-else clauses